velox/exec/Spiller.cpp

```diff
@@ -165,6 +165,13 @@ Spiller::Spiller(
       "facebook::velox::exec::Spiller", const_cast<HashBitRange*>(&bits_));
 
   VELOX_CHECK_EQ(container_ == nullptr, type_ == Type::kHashJoinProbe);
+  VELOX_CHECK_EQ(
```

Comment:
```
if (rowType_->size() != container_->columnTypes().size()) {
  VELOX_FAIL(); with detailed error message like the actual types of container_->columnTypes()?
}
```
I am not sure if VELOX_CHECK_EQ build error message before the check or only after the check fails.

Reply:
VELOX_CHECK_EQ will build the message in the same way VELOX_FAIL does, no need for if() statement.